src/uniq.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Returns an array with all duplicate values removed.
```

Comment:
Maybe good to be even more explicit and saying that it will return a new/copy of the array with the duplicate values removed?